lib/components/sidebar-component.js

```diff
@@ -89,7 +89,9 @@ export default class SidebarComponent extends AsyncBaseContainer {
 		const searchSelector = By.css( '.site-selector input[type="search"]' );
 		const siteSelector = By.css( `.site-selector .site a[aria-label*="${ searchString }"]` );
 
-		const searchEnabled = await driverHelper.isElementPresent( this.driver, searchSelector );
+		const searchElement = await this.driver.findElement( searchSelector );
```

Comment:
The element is still in the DOM, but hidden. The previous approach of using whether the element exists didn't work